app/bundles/ApiBundle/Controller/CommonApiController.php

```diff
@@ -1220,7 +1220,7 @@ protected function processForm($entity, $parameters = null, $method = 'PUT')
      *
      * @return Response|array
      */
-    protected function returnError($msg, $code = Codes::HTTP_OK, $details = [])
+    protected function returnError($msg, $code = Codes::HTTP_INTERNAL_SERVER_ERROR, $details = [])
```

Comment:
Wouldn't 400 Bad Request make more sense here? If the code ran it's not really a 500, and if the code fails, a 500 will be returned by the web server anyway, but the code should never fail under reasonable circumstances.